File: psup assignment/java/q5.java
import java.util.*;
class PassCheck {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter marks: ");
        int marks = input.nextInt();
        if (marks >= 40) {
            System.out.println("The student has passed.");
        }
        input.close();
    }
}

File: psup assignment/cpp/q5.cpp
#include <iostream>
#include <string>
#include <cctype>
int main() {
    int marks;
    std::cout << "\n--- Program 5: Pass Mark Check ---\n";
    std::cout << "Enter marks: ";
    std::cin >> marks;
    if (marks >= 40) {
        std::cout << "The student has passed." << std::endl;
    }
    return 0;
}